example/android/app/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepsdk_example/MyApplication.java

```diff
@@ -23,7 +24,8 @@ public void onCreate() {
             Lifecycle.registerExtension();
             Signal.registerExtension();
             Assurance.registerExtension();
-            MobileCore.start(o -> MobileCore.configureWithAppID("yourAppId"));
+            Consent.registerExtension();
+            MobileCore.start(o -> MobileCore.configureWithAppID(""));
```

Comment:
Also what is the `o` parameter for MobileCore.start API. Lets replace with `_` or a descriptive name.